contents/handbook/company/fuzzy-ownership.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Fuzzy ownership
+sidebar: Handbook
+showTitle: true
+---
+
+As we continue to grow, it can be hard to figure out who is the owner of something at PostHog. This is especially difficult if you are new to the team and don’t have a lot of historic context.
+
+There are also some things that don’t have an owner at all, so we've created a simple process to deal with these that you might find helpful. Ideally, we want the default assumption to be to _not_ a) hire a new person, or b) escalate to James/Tim.
+
+## Figuring out who owns a thing at PostHog
+
+An owner can be a single person or a small team - either is fine. There are two placees you can figure out who owns something at PostHog:
+
+- Product features - see the [Feature ownership page](/handbook/engineering/feature-ownership)
+- Anything non-product - visit the relevant [small team page](/teams)
+
+If you spot anything out of date or not obviously clear, please raise a PR!
+
+## Figuring out a new owner for a thing we’ve identified
+
+- First, raise that it doesn’t have an owner however you want. For example, you might add ‘Settings page’ to the Feature Ownership table but without an owner, then flag the PR in Slack.
+- Ideally someone just puts their hand up and says ‘I’ll do it’. This can be for a fixed period, until something happens (e.g. new hire joins, X months elapse), or indefinitely.
+- If no one puts their hand up because it’s tricky/not obvious/everyone is super busy, the relevant people that touch the thing should decide between them. These may be team leads, but not necessarily. 
+  - To make it feel less like ‘this is your job forever now’ you could say ‘X will be the owner of this thing for Y period, after which Z should happen.’
+  - If you can’t work it out between team members/leads, ask a relevant person in Exec who can help be tiebreaker.
+- If you/your team becomes the owner in a _temporary_ way, part of your job as owner is to figure out the long term plan for the thing
+- If you’re struggling to figure out how to prioritize the new thing vs. other work, ask your team/manager for advice.
```

Comment:
```suggestion
- First, raise that it doesn’t have an owner however you want. For example, you might add ‘Settings page’ to the Feature Ownership table but without an owner, then flag the PR in Slack.

- Ideally someone just puts their hand up and says ‘I’ll do it’. This can be for a fixed period, until something happens (e.g. new hire joins, X months elapse), or indefinitely.

- If no one puts their hand up because it’s tricky/not obvious/everyone is super busy, the relevant people that touch the thing should decide between them. These may be team leads, but not necessarily. 
  - To make it feel less like ‘this is your job forever now’ you could say ‘X will be the owner of this thing for Y period, after which Z should happen.’
  - If you can’t work it out between team members/leads, ask a relevant person in Exec who can help be tiebreaker.
  
- If you/your team becomes the owner in a _temporary_ way, part of your job as owner is to figure out the long term plan for the thing.

- If you’re struggling to figure out how to prioritize the new thing vs. other work, ask your team/manager for advice.
```